integration/src/main/java/scripts/testscripts/SnapshotDatasetOperations.java

```diff
@@ -132,7 +133,7 @@ private void previewUserJourney(CatalogClient client) throws ApiException {
     @SuppressWarnings("unchecked")
     Map<String, String> row = (Map<String, String>) sampleTable.getRows().get(0);
 
-    assertThat(row, hasEntry(is("sample_id"), is("sample5")));
+    assertThat(row, hasEntry(is("sample_id"), is(notNullValue())));
```

Comment:
This change is because "sample5" exact matching is dependent on the dataset that is created in the environment, not on our test setup. We want to make sure there is some data there, but I don't think we actually need to test _exactly_ what it is, and this makes the test less brittle.